configure.ac

```diff
@@ -595,6 +597,7 @@ CFLAGS+=" -std=c99 -Werror=implicit-int -Werror=implicit-function-declaration -W
 
 # At this point, CFLAGS is set to something sensible
 AC_PROG_CC
+AX_PROG_CC_FOR_BUILD
```

Comment:
I did not need to install anything to use this. So looks like the distros include this by default?

Reply:
Although I think we should be allowed to include this file into our package, I added a note to README.md about how to install it or where to download it in case it is not available anyway. I case this causes trouble with distributors later on, we can still decide to include it into the m4 directory.

Reply:
also added it a s build requirement to the example RPM spec file.